lib/scraper/scraper_base.rb

```diff
@@ -6,10 +6,14 @@
 module Scraper
   class ScraperBase
 
+    EVENT_ATTRS = [
+      'browser_url', 'origin_system', 'title', 'description', 'start_date', 'end_date', 'free', 'featured_image_url', 'identifiers'
+    ].freeze
+    
```

Comment:
If you want to move this to the base, that's fine, but it's still in your resistance calendar class and here: https://github.com/RagtagOpen/cta-scraper/blob/8ea38c9e86ceb470425691bab66d87d946d4fc0c/lib/scraper/emilys_list.rb#L9-L11.

Reply:
thanks--removed